griptape/utils/chat.py

```diff
@@ -16,22 +17,39 @@ class Chat:
     prompt_prefix: str = field(default="Q: ", kw_only=True)
     response_prefix: str = field(default="A: ", kw_only=True)
     output_fn: Callable[[str], None] = field(
-        default=Factory(lambda: print), kw_only=True
+        default=lambda x: print(x, end=""), kw_only=True
     )
+    stream: bool = field(default=False, kw_only=True)
+
+    def start(self, initial_turn: str = None) -> None:
+        if initial_turn:
+            self.output_fn(initial_turn + "\n")
+
+        self.structure.stream = self.stream
+        self.structure.prompt_driver.stream = self.stream
```

Comment:
Is this necessary? The `Stream` utility should take care of adding this already.

Reply:
I had thought about that. If we remove this then we'll require that the user add stream=True to the prompt_driver and the Agent definition. The request had asked for stream to be a parameter to be passed to Chat. 
To keep it as a parameter in Chat, should I create a deepcopy of the Agent and mutate the local copy or remove the stream parameter from Chat?

Reply:
Yeah we had gone back and forth on this but ultimately decided not to mutate.  Let's keep it consistent with `Stream` for now. The docs are actually incorrect on this point, fix incoming soon.